lesson_16/Test_for_hw_16.py

```diff
@@ -0,0 +1,46 @@
+import unittest
+from homework_16 import SiteUser
+
+class homework_16_SiteUser_test(unittest.TestCase):
+
+    def setUp(self) :
+       self.Johny = SiteUser ("Johny Silverhand", "samurai1988@aft.com", "moderator")
+       self.Adam = SiteUser("Adam Smasher", "smasher@arasaka.com", "admin")
+       self.Jack = SiteUser ("Jack Welles", "Jaquito@araska.com", "user")
+
+    def test_1_baselogcount(self):
+       AR = self.Johny.logcount
+       ER = 0
```

Comment:
```suggestion
       AR = self.Johny.logcount
       ER = 0
```
тут дві проблеми - перше - не треба скорочувати, у нас жеж не папір,
друге - капсом принято задавати назви СТАТИЧНИХ данних, тобто змінні, що не змінюються

Reply:
Спасибо, учту